src/transformers/models/bart/modeling_flax_bart.py

```diff
@@ -405,7 +405,7 @@ def setup(self) -> None:
             num_heads=self.config.encoder_attention_heads,
             dropout=self.config.attention_dropout,
         )
-        self.self_attn_layer_norm = nn.LayerNorm(dtype=self.dtype)
+        self.self_attn_layer_norm = nn.LayerNorm(dtype=self.dtype, epsilon=1e-05)
```

Comment:
@patil-suraj - the default in PyTorch is 1e-05, so I adapted it for all Bart-like models. Given that PT and Flax tests were passing for Bart before I think this "bug correction" is fine in terms of backwards compatibility